### DISCUSSION TITLE
How do I stop OMZ from showing errors in my console? It's preventing multiple ansible roles from running successfully.

### DISCUSSION THREAD

**Discussion by CodeBradley** [ohmyzsh/ohmyzsh#12598] [Help]
This is the error I keep getting when running `ansible.builtin.shell`. This has happened a number of times and I'm not the package maintaner in some of these ansible roles. It's also ensuring that .zshrc is configured correctly by confirming that this command runs in ZSH itself. 

The plugin it's mentioning is not installing because NPM isn't installed, but NPM isn't installing because OMZ is showing this error in the console.

`fatal: [he]: FAILED! => {"changed": true, "cmd": "/usr/bin/zsh -ic \"nvm --version\"", "delta": "0:00:00.194864", "end": "2024-08-05 17:20:36.629696", "msg": "non-zero return code", "rc": 127, "start": "2024-08-05 17:20:36.434832", "stderr": "zsh:1: command not found: nvm", "stderr_lines": ["zsh:1: command not found: nvm"], "stdout": "[oh-my-zsh] plugin 'zsh-history-enquirer' not found", "stdout_lines": ["[oh-my-zsh] plugin 'zsh-history-enquirer' not found"]}`

Also, I set OMZ update to auto and I still get a console notice for that. How do I shut that off?

**Comment by mcornella** [comment]
You need to review the order of installation of the required tools. The error says that nvm is not installed, so probably it's not installed or not loaded at the right time (it puts loading lines at the bottom of the zshrc file). The same for the other plug-in, it should be installed properly according to ohmyzsh docs.
